cmd/containerd/builtins_windows.go

```diff
@@ -0,0 +1,25 @@
+// +build windows
+
+/*
+Copyright The containerd Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	_ "github.com/containerd/containerd/diff/lcow"
```

Comment:
I'm fine with this but since you are only doing WCOW process for the first round we dont need any lcow/ stuff

Reply:
The problem is that the `lcow` differ is required here. https://github.com/containerd/containerd/blob/4e6e61c44e83ba1f69aa37ea603a9f23383821fb/services/diff/service_windows.go#L22

Reply:
Oh funny. No problem just leave it. I dont think its worth the tag it wont work here anyways